ajax/libs/ryejs/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "ryejs",
+  "filename": "rye.min.js",
+  "version": "0.1.2",
+  "description": "A lightweight browser library",
+  "keywords": [
+    "lightweight",
+    "ES5",
+    "DOM",
+    "browser",
+    "modern"
+  ],
+  "homepage": "http://ryejs.com/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ryejs/rye.git"
+  },
+  "license": "MIT",
```

Comment:
The license in github is different from npm.
@Piicksarn You need to confirm that with the author.